src/chrome/content/rules/Casual-Effects.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Casual-Effects.com">
+	<target host="casual-effects.com" />
+	<target host="www.casual-effects.com" />
+
```

Comment:
```xml
	<securecookie host=".+" name=".+" />
```

Reply:
This is the first time I have encountered this while contributing rulesets. So that I can write better rulesets in the future, how does one tell when the `securecookie` tag is required?

Reply:
Unless the website breaks with this rule, you should usually add it.